main.go

```diff
@@ -120,7 +120,7 @@ func main() {
 	// Cert won't be ready until manager starts, so start a goroutine here which
 	// will block until the cert is ready before setting up the controllers.
 	// Controllers who register after manager starts will start directly.
-	go setupControllers(mgr, cCache, queues, certsReady, cfg.ManageJobsWithoutQueueName)
+	go setupControllers(mgr, cCache, queues, certsReady, cfg.ManageJobsWithoutQueueName, waitForPodsReady(&cfg))
```

Comment:
Before this line grows too much:

```suggestion
	go setupControllers(mgr, cCache, queues, certsReady, cfg)
```

Reply:
This was not addressed.

Reply:
Ups, going to address now.

Reply:
Done